### DISCUSSION TITLE
How to change the comment box html?

### DISCUSSION THREAD

**Discussion by ghost** [posativ/isso#957] [Q&A]
I want to delete the preview button and some parts of the comment box. How do I do this? Thanks!

**Comment by ix5** [comment]
You could either:

a) hide the parts you don't want shown with CSS or
b) edit the relevant parts of the `postbox` template Javascript (see below) and re-compile the `comment.min.js` via `make js`

https://github.com/posativ/isso/blob/64374a8af978f1cfd4a91875c18a309122c7d9ca/isso/js/app/templates/postbox.js#L38-L40
